include/xtensor/xchunked_array.hpp

```diff
@@ -331,6 +334,18 @@ namespace xt
         return xchunked_array<chunk_storage, EXT>(chunk_storage(), std::forward<S>(shape), std::forward<S>(chunk_shape), chunk_memory_layout);
     }
 
+    template <class T, layout_type L, class EXT, class S>
+    xchunked_array<xarray<xarray<T>>, EXT> chunked_array(std::initializer_list<S> shape, std::initializer_list<S> chunk_shape, layout_type chunk_memory_layout)
+    {
+        using sh_type = std::vector<std::size_t>;
+        sh_type sh = xtl::make_sequence<sh_type>(shape.size());
+        std::copy(shape.begin(), shape.end(), sh.begin());
+        sh_type ch_sh = xtl::make_sequence<sh_type>(chunk_shape.size());
+        std::copy(chunk_shape.begin(), chunk_shape.end(), ch_sh.begin());
```

Comment:
You can use `xtl::forward_sequence` and forward the call to the generic implementation of `chunked_array`, similarly to what you did in `xtensor-io`.